src/Security/Acl/Permission/AdminPermissionMap.php

```diff
@@ -111,6 +111,11 @@ class AdminPermissionMap implements PermissionMapInterface
         ],
     ];
 
+    /**
+     * @todo Remove phpDoc when this PR https://github.com/symfony/security-acl/pull/58 is merged
```

Comment:
It is merged, so I suppose the phpdoc can be removed, or rather, that this change should be made:
```suggestion
     * @todo Remove phpDoc when this PR https://github.com/symfony/security-acl/pull/58 is released
```

Reply:
I think it is already released, can you remove this comment and update to 3.1 on composer.json?

https://github.com/symfony/security-acl/releases/tag/v3.1.0